jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/Request.java

```diff
@@ -507,7 +507,15 @@ default Transport getTransport()
      * @param listener the listener that receives response events
      */
     void send(Response.CompleteListener listener);
-
+    
+    /**
+     * <p>Sends this request and asynchronously notifies the given listener for response events 
+     * parse content with requested file and save it into {@link Path}</p>
+     *
+     * @param listener the listener that receives response events
+     */
+    void send(PathResponseListener listener);
```

Comment:
This method is not necessary, please remove it.
`PathResponseListener` _is-a_ `CompleteListener`, so the existing method will work.

Reply:
Indeed, `PathResponseListener` implements `CompleteListener` so it is not needed to have method overloading for `PathResponseListener`
My fail!